lib/vagrant/util/downloader.rb

```diff
@@ -245,6 +247,9 @@ def options
         options << "-u" << @auth if @auth
         options << "--location-trusted" if @location_trusted
 
+        options.concat(
+          Vagrant::Util::MapCommandOptions.map_to_command_options(@extra_download_options))
```

Comment:
Another thing we should do with this config is call this conversion map in the `finalize!` method of `#Config::Vm` instead of here. Then we could give people some warning in the config validate step in case their command options are wrong before they go and use them, and just pass along the `@extra_download_options` here since it'll already be converted. This will also make it easier to use else where if we have to since it'll already be the right format.